delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/ResourcesImpl.scala

```diff
@@ -44,7 +44,9 @@ final class ResourcesImpl private (
     for {
       projectContext             <- fetchContext.onCreate(projectRef)
       schemeRef                  <- expandResourceRef(schema, projectContext)
-      (iri, compacted, expanded) <- sourceParser(projectRef, projectContext, source)
+      (iri, compacted, expanded) <- sourceParser(projectRef, projectContext, source).map { j =>
+                                      (j.iri, j.compacted, j.expanded)
+                                    }
```

Comment:
Using the JsonLdResult unapply could be an alternative though it can be looked at later